src/PhpPact/Standalone/StubService/Service/StubServerHttpService.php

```diff
@@ -53,7 +53,7 @@ public function healthCheck(): bool
         $body = $response->getBody()->getContents();
 
         if ($response->getStatusCode() !== 200
-            || $body !== "Stub service running\n") {
+            || $body !== "Mock service running\n") {
```

Comment:
Not sure why it must be `Mock`, not `Stub`. Maybe because the header `X-Pact-Mock-Service` above. I tried to changed to `X-Pact-Stub-Service` but this header doesn't exist. Here is the error I got:

```
1) PhpPactTest\Standalone\StubServer\Service\StubServerHttpServiceTest::testHealthCheck
GuzzleHttp\Exception\ServerException: Server error: `GET http://localhost:7201/` resulted in a `500 Internal Server Error` response:
{"message":"No interaction found for GET /","interaction_diffs":[]}
```